tools/build_variables.bzl

```diff
@@ -383,6 +383,7 @@ libtorch_distributed_extra_sources = [
     "torch/csrc/distributed/rpc/tensorpipe_agent.cpp",
     "torch/csrc/distributed/rpc/tensorpipe_utils.cpp",
     "torch/csrc/distributed/rpc/testing/faulty_process_group_agent.cpp",
```

Comment:
Shall we remove `faulty_process_group_agent` and `process_group_agent` in followup PRs? Thanks!

Reply:
Yep! Will do that and update code comments